Content.Shared/Chat/V2/Messages.cs

```diff
@@ -0,0 +1,299 @@
+﻿using Robust.Shared.Serialization;
```

Comment:
SO MANY MESSAGES

This is a lot of the skeleton but not all: there'll be more messages on the egress side :)